### PR TITLE
add disputes to Kusama runtime

### PR DESCRIPTION
**Pull request by rphmeier**
skip check-dependent-cumulus

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -1213,6 +1213,13 @@ impl parachains_initializer::Config for Runtime {
 	type WeightInfo = weights::runtime_parachains_initializer::WeightInfo<Runtime>;
 }
 
+impl parachains_disputes::Config for Runtime {
+	type Event = Event;
+	type RewardValidators = ();
+	type PunishValidators = ();
```
Comment (shawntabrizi) [inline runtime/kusama/src/lib.rs:1219]: is there a TODO to populate this?

Comment (rphmeier) [reply runtime/kusama/src/lib.rs:1219]: Yes

Comment (rphmeier) [reply runtime/kusama/src/lib.rs:1219]: @ordian is working on it

**Review by drahnr — APPROVED** [review]

File: runtime/kusama/src/lib.rs (hunk 1 above)
Comment (ordian) [reply runtime/kusama/src/lib.rs:1219]: These changes are probably going to land in 0.9.14. Not blocking this PR.

**Pull request merged**